static/boosthub-preload.js

```diff
@@ -66,6 +66,35 @@ function convertHtmlStringToPdfBlob(htmlString, printOptions) {
     })
   })
 }
+
+function findInPage(text, options) {
+  const webContents = remote.getCurrentWebContents()
+  if (webContents) {
+    return webContents.findInPage(text, options)
+  }
+}
+
+function stopFindInPage(action) {
+  const webContents = remote.getCurrentWebContents()
+  if (webContents) {
+    return webContents.stopFindInPage(action)
+  }
+}
+
+function addWebContentsEvent(eventName, callback) {
+  const webContents = remote.getCurrentWebContents()
+  if (webContents) {
+    return webContents.on(eventName, callback)
```

Comment:
We should not have this method. Please make one just for 'found-in-page' and we should not expose all arguments to the callback. See `convertHtmlStringToPdfBlob` method in this file. It only listens `did-finish-load` and doesn't expose any arguments to its callback.

Reply:
Okay, will do